modules/control/controller/lat_controller.cc

```diff
@@ -409,9 +409,9 @@ Status LatController::ComputeControlCommand(
   if (VehicleStateProvider::Instance()->linear_velocity() <
```

Comment:
Save the Instance() to a temp variable and make the layout easier.

Reply:
Done